web/source/osk/inlinedOskView.ts

```diff
@@ -0,0 +1,182 @@
+// Includes KMW-added property declaration extensions for HTML elements.
+/// <reference path="../kmwexthtml.ts" />
+// Includes the touch-mode language picker UI.
+/// <reference path="languageMenu.ts" />
+/// <reference path="lengthStyle.ts" />
+// Defines desktop-centric OSK positioning + sizing behavior
+/// <reference path="layouts/targetedFloatLayout.ts" />
+/// <reference path="oskView.ts" />
+
+/***
+   KeymanWeb 10.0
+   Copyright 2017 SIL International
+***/
+
+namespace com.keyman.osk {
+  type OSKPos = {'left'?: number, 'top'?: number};
+
+  export class InlinedOSKView extends OSKView {
+    // Key code definition aliases for legacy keyboards  (They expect window['keyman']['osk'].___)
+    modifierCodes = text.Codes.modifierCodes;
+    modifierBitmasks = text.Codes.modifierBitmasks;
+    stateBitmasks = text.Codes.stateBitmasks;
+    keyCodes = text.Codes.keyCodes;
+
+    public constructor(modeledDevice: utils.DeviceSpec, hostDevice?: utils.DeviceSpec) {
+      super(modeledDevice, hostDevice);
+
+      this.activationMode = ActivationMode.manual;
+    }
+
+    public get element(): HTMLDivElement {
+      return this._Box;
+    }
+
+    /**
+     * Function     _Unload
+     * Scope        Private
+     * Description  Clears OSK variables prior to exit (JMD 1.9.1 - relocation of local variables 3/9/10)
+     */
+    _Unload() {
+      this.keyboardView = null;
+      this.bannerView = null;
+      this._Box = null;
+    }
+
+    protected setBoxStyling() {
+      const s = this._Box.style;
+      s.display  = 'none';
+      // Positioned with no relative offset from its default position.
+      // This allows _Box to still serve as an offsetParent for keytip & subkey menu positioning.
+      s.position = 'relative';
+    }
+
+    protected postKeyboardLoad() {
+      this._Visible = false;  // I3363 (Build 301)
+
+      this._Box.onmouseover = this._VKbdMouseOver;
+      this._Box.onmouseout = this._VKbdMouseOut;
+
+      if(this.displayIfActive) {
+        this.present();
+      }
+    }
+
+    /**
+     * Function     restorePosition
+     * Scope        Public
+     * @param       {boolean?}      keepDefaultPosition  If true, does not reset the default x,y set by `setRect`.
+     *                                                   If false or omitted, resets the default x,y as well.
+     * Description  Move OSK back to default position, floating under active input element
+     */
```

Comment:
Could you fixup all the javadoc-style comments to be correct javadoc structure? (No need to put types in the comment, because TS does that for us.)